editor/js/commands/SetValueCommand.js

```diff
@@ -1,5 +1,39 @@
 import { Command } from '../Command.js';
 
+function setValue( object, path, value ) {
```

Comment:
This creates an unnecessary complexity in `SetValueCommand`. I've filed a different PR that solves the issue more consistent so light shadows are treated like geometries and materials, see #28608.

Reply:
good